3.6/pkg_installer.R

```diff
@@ -0,0 +1,22 @@
+docker_pkg_install <- function(package_csv) {
+  pkgs <- read.csv(package_csv, stringsAsFactors = FALSE)
+
+  utils::install.packages("parallel")
+
+  cores <- max(1, parallel::detectCores() - 1)
+
+  print("Trying on cores:")
+  print(cores)
+
+  try_package <- function(package, ...) {
+    tryCatch({
+      # TODO: Do we need a particular install library?
```

Comment:
Do we need to specify a particular library @colearendt? Or will this automatically go to the right place because it's being called with the R version.